extensions/wikia/ArticleVideo/bower_components/skin-config/validate.js

```diff
@@ -0,0 +1,71 @@
+var fs = require( 'fs' );
+var stripComments = require( 'strip-json-comments' )
+
+// assumes this is run via node.
+function usage( message ) {
+    if( message ) {
+        console.log( message );
+    }
+    console.log( "USAGE: " + process.argv[1] + " <path-to-schema> <path-to-json>" );
+}
+
+// unclear to me yet which library to bet on, if not something else entirely.
+
+function validate_with_jsonschema( schema_object, json_object ) {
+    var Validator = require('jsonschema').Validator;
+    var v = new Validator();
+    var r = v.validate( json_object, schema_object );
+    if( r.errors && r.errors.length ) {
+        var err = "(jsonschema) VALIDATION FAILED: " + r.errors;
+        throw err;
+    }
+}
+
+function validate_with_tv4( schema_object, json_object ) {
+    var tv4 = require( 'tv4' );
+    var validation_result = tv4.validate( json_object, schema_object );
+    if( ! validation_result ) {
+        var err = "(tv4) VALIDATION FAILED: " + JSON.stringify( tv4.error );
+        throw err;
+    }
+}
+
+function validate( schema_object, json_object ) {
+    validate_with_jsonschema( schema_object, json_object );
+    validate_with_tv4( schema_object, json_object );
+    console.log( "Passed." );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)